network/node/genesis/custom_genesis.go

```diff
@@ -18,6 +19,44 @@ type CustomGenesis struct {
 	ForkConfig *CustomGenesisForkConfig `json:"forkConfig"`
 }
 
+func HandleAdditionalFields(raw *map[string]interface{}) {
```

Comment:
I wonder if this can be modified and used as a custom marshaller ?

Reply:
(Not really required for this PR, just a pretty-up)

Reply:
Let me look into this, thanks